### ISSUE TITLE
Change Twelve of Code colors to CSS classes for when mentioned outside of the formatting classes

### ISSUE THREAD

**Issue by Mesure73L** [Mesure73L/mesure.x10.mx#66]
### What problem were you having?

You have to change it everywhere instead of just in one place.

### What solution would you like?

Use CSS variables.

### What alternatives have you considered?

Keep changing it everywhere.

### What are some pros and cons of doing this?

Pros: Easier to change colors
Cons: IDK

### What page is this for?

Twelve of Code

**Comment by BeepBot99** [comment]
WOW

**Comment by BeepBot99** [comment]
I literally started doing this 3 weeks ago and you told me to switch it back

**Comment by Mesure73L** [comment]
oh wait the bot isn't here /t

**Comment by BeepBot99** [comment]
@Mesure73L #68

**Comment by Mesure73L** [comment]
> @Mesure73L #68

Please don't mention me in two separate places about the same thing. I see everything even if you don't mention me.

**Comment by BeepBot99** [comment]
Ok, sorry.

**Comment by BeepBot99** [comment]
I forgot that I had mentioned you there when I mentioned you here.

**Comment by BeepBot99** [comment]
@Mesure73L I did this for the modified `style.css` in the challenges repo.

**Comment by BeepBot99** [comment]
I'm switching this to 1.0.0, you can assign me instead of you want or you can still do it.

**Comment by BeepBot99** [comment]
@Mesure73L Do you want me to do this? That would make #156 A lot easier.

**Comment by Mesure73L** [comment]
> @Mesure73L Do you want me to do this? That would make #156 A lot easier.

Sure, thanks!

**Comment by Mesure73L** [comment]
Please do variables for everything that the creator of a theme pack might possibly want to change, not just colors.

**Comment by BeepBot99** [comment]
What about like sizes? Should those change or not? Also, should theme packs automatically be applied to challenge CSS too? In that case, it would not be a separate CSS files for challenges.
Because alternatively, I was thinking that the challenge CSS file could be optional, and the creator of a challenge could make their own if they wanted to.

**Comment by Mesure73L** [comment]
They should be applied to challenge CSS too.

**Comment by Mesure73L** [comment]
Prism CSS should also be changeable. If you need any help with this I can do part of it.

**Comment by BeepBot99** [comment]
Yeah, I know how to do Prism CSS because I've used it before.

**Comment by Mesure73L** [comment]
Those two sentences weren't related, but I also know how to use Prism CSS, it's not that complicated.

**Comment by BeepBot99** [comment]
Okay, I'm doing this, now STFU.

**Comment by BeepBot99** [comment]
(sry for being rude btw)

**Comment by Mesure73L** [comment]
You call that rude?

**Comment by BeepBot99** [comment]
Yes, I do in fact call that rude. 🤯 Please learn what rude means in the future.

**Comment by Mesure73L** [comment]
You're the one who needs to learn what rude means. 🤯

**Comment by BeepBot99** [comment]
What do you mean? No you are.

**Comment by Mesure73L** [comment]
??? No, you are.

**Comment by BeepBot99** [comment]
Thank you for saying I'm kind.
![image](https://github.com/twelve-of-code-official/website/assets/74377508/afc7c0d1-bd7a-4f65-9450-03c69fec30e1)

**Comment by Mesure73L** [comment]
I did not say you are kind. I said I am kind.

**Comment by Mesure73L** [comment]
What are you deleting?

**Comment by BeepBot99** [comment]
![image](https://github.com/twelve-of-code-official/website/assets/74377508/a11678a7-7ac8-42e5-a168-a18010a0533c)

**Comment by Mesure73L** [comment]
![image](https://github.com/twelve-of-code-official/website/assets/115181664/861b584a-bb61-4bdc-9006-7f9194398423)

**Comment by BeepBot99** [comment]
Yes, that is what I said. By the way, thank you.
![image](https://github.com/twelve-of-code-official/website/assets/74377508/d978d583-dd4f-4979-aeb3-7089e2144ca4)

**Comment by Mesure73L** [comment]
![image](https://github.com/twelve-of-code-official/website/assets/115181664/6dd51d3c-6da1-4472-bef3-541503dd02e9)
![image](https://github.com/twelve-of-code-official/website/assets/115181664/ce39ae02-6afc-47bc-8c0f-056c26139682)

**Comment by Mesure73L** [comment]
Stop being stupid or I'll remove you from this organization.

**Comment by BeepBot99** [comment]
And how exactly do you plan on doing that?

**Comment by Mesure73L** [comment]
Put me back to owner.

**Comment by Mesure73L** [comment]
Or I'll remove the FTP accounts.

**Comment by Mesure73L** [comment]
(Which is something you can't change)

**Comment by BeepBot99** [comment]
If you promise to never threaten to remove me from the organization or remove my owner role or do either of those ever again. Also I can make a new domain with new FTP accounts (I won't do that though).

**Comment by Mesure73L** [comment]
Fine, but don't abuse your power and **FOLLOW THE CODE OF CONDUCT**

**Comment by Mesure73L** [comment]
Which, by the way, you've broken multiple times.

**Comment by Mesure73L** [comment]
Put me back.

**Comment by Mesure73L** [comment]
@BeepBot99 Put me back as owner or I will actually remove the FTP accounts.

**Comment by Mesure73L** [comment]
But jsyk if you're breaking the CoC as an owner it doesn't look good, so please stop breaking it. I made them for a reason, so follow them. Keep everything kind and respectful. If I edit your message because from a disrespectful message to a more kind one, please do not edit it back.

**Comment by Mesure73L** [comment]
@BeepBot99

**Comment by BeepBot99** [comment]
Chill I wasn't online.

**Comment by BeepBot99** [comment]
Done.
